docs/running-on-mesos.md

```diff
@@ -734,6 +734,36 @@ See the [configuration page](configuration.html) for information on Spark config
   </td>
   <td>2.1.0</td>
 </tr>
+<tr>
+  <td><code>spark.mesos.dispatcher.queue</code></td>
+  <td><code>(none)</code></td>
+  <td>
+    Set the name of the dispatcher queue to which the application is submitted.
+    The specified queue must be added to the dispatcher with <code>spark.mesos.dispatcher.queue.[QueueName]</code>.
+    If no queue is specified, then the application is submitted to the "default" queue with 0.0 priority.
+  </td>
```

Comment:
It seems that the last version column is missing. Could you add the following?
```
<td>3.1.0</td>
```

Reply:
Done